java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Fixed the CSV download issue
```

Comment:
This will be read by users. A more detailed message would be helpful. If there is a bug report you can link to it

Reply:
Is the GitHub issue considered a bug report? I saw bsc#... being linked in other entries but I don't know what it stands for.